src/main/java/xyz/nucleoid/plasmid/mixin/game/rule/HungerManagerMixin.java

```diff
@@ -0,0 +1,28 @@
+package xyz.nucleoid.plasmid.mixin.game.rule;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.entity.player.HungerManager;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.util.ActionResult;
+import xyz.nucleoid.plasmid.game.manager.GameSpaceManager;
+import xyz.nucleoid.plasmid.game.rule.GameRuleType;
+
+@Mixin(HungerManager.class)
+public class HungerManagerMixin {
+    @Inject(method = "update", at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/player/PlayerEntity;heal(F)V", shift = At.Shift.BEFORE, ordinal = 0), cancellable = true)
```

Comment:
This injection point prevents the other branches of regeneration code from executing and thus will prevent the player from regenerating. The mixin added in NucleoidMC/Cake-Wars@55f903d9d1ae12d8ca420485510059b250f39dcf should be used instead.

Reply:
Fixed in 7d577c007da59c7f0c5c333e308037c881002d6e